src/main/java/com/google/sps/DatastoreGarbageCollector.java

```diff
@@ -0,0 +1,78 @@
+package com.google.sps;
+
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.time.Instant;
+import java.time.Duration;
+import java.util.Date;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.PropertyFilter;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.Timestamp;
+
+public class DatastoreGarbageCollector {
+    private ScheduledExecutorService ses;
+
+    private static void deleteOldEvents() {
+    System.out.println("Deleting events that have occurred more than 1 day in the past");
```

Comment:
nit: can you indent this lines 22-40?

Actually, what may be better is to use a code formatter. Here's [one](https://github.com/google/google-java-format) I was able to easily download and use. It abides by Google Java Style Guide. Here are the steps to download and run


1. Run `wget https://github.com/google/google-java-format/releases/download/v1.15.0/google-java-format-1.15.0-all-deps.jar` to download the jar file. No need to download into repo just have it locally on your machine.
2. Run `java -jar google-java-format-1.15.0-all-deps.jar --replace <filename>` to format the java file.

Reply:
Thanks, this is handy!